mkosi.md

```diff
@@ -194,6 +194,16 @@ details see the table below.
   Linux, additional repositories must be passed in the form `<name>::<url>`
   (e.g. `myrepo::https://myrepo.net`).
 
+`--extra-repositories=`
+
+: Additional package repository files to be used during installation.
+  Expects one or more paths to repository files, separated by commas.
+  This option may be used multiple times in which case the list of
+  repository files to use is combined. Use "!\*" to remove all
+  repository files from to the list or use "!repo-file" to remove
+  just one specific repository file. Currently only supported for
+  Fedora, Mageia, CentOS, Photon, OpenMandriva.
```

Comment:
My assumption is, because it's already supported in the logic of `ListAction`. The whole CLI is from before I started looking at mkosi and it's quite a heavy beast. Just a couple of minutes ago, I learned that `[a, b, c]` lists are supported in the config files and I haven't yet looked whether this is even documented.

Reply:
I mostly copied the documentation from the `repositories` argument.
I believe it's meant to be used in commandline arguments to remove list elements defined in the `mkosi.default` file.